pkg/module/temp_files.go

```diff
@@ -0,0 +1,65 @@
+package module
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+var tempFiles []*os.File
```

Comment:
We should definitely revisit our lint settings, since you are introducing here a global variable for one purpose to a huge package. I would prefer not using a global variable in a pkg in general, if it is not for testing purposes.

Please revisit and try to find a way to not use a global variable here, maybe by introducing a struct or solve it using input parameters.

Some articles to read: 
- https://medium.com/higher-order-functions/golang-its-a-global-problem-1a939461c16d
- https://dave.cheney.net/2017/06/11/go-without-package-scoped-variables

Reply:
packaged the variable in a struct